app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -136,7 +136,7 @@
       = link_to(admin_conference_roles_path(@conference.short_title)) do
         %span.fa.fa-group
         Roles
-  - if can? :index, PaperTrail::Version.new(item_type: 'User')
+  - if can?(:index, PaperTrail::Version.new(conference_id: @conference.id, item_type: 'Event')) || can?(:index, PaperTrail::Version.new(conference_id: @conference.id, item_type: 'Registration'))
```

Comment:
No.
conference_id: @conference.id part is so that the link appears only if in conferences that the user has access to.Without this part, the Revision Histroy(conference sidebar) link will appear for all conferences if user has organizer/cfp/info_desk role in atleast 1 conference.

Again `can?(:index, PaperTrail::Version.new(conference_id: @conference.id)` is not enough
as this will make the link appear only for organizers.

Organizers have :index permission for all versions that for which the user is an organizer and 'User'. Cfp users have :index permission only in Event EventType Track DifficultyLevel EmailSettings Room Cfp Program Comment versions of confs that they have cfp_role
and Info Desk users have access to Registration type versions
So `can?(:index, PaperTrail::Version.new(conference_id: @conference.id, item_type: 'Event'))` will authorize organizers and cfp team users and `can?(:index, PaperTrail::Version.new(conference_id: @conference.id, item_type: 'Registration')` will authorize info_desk users